File: dz/s.java
import java.util.Stack;

public class s {
    public static void main(String[] args) {
        // стек
        Stack<Integer> stack = new Stack<>();
        for (int i = 0; i < 5; i++) {
            stack.push(i);
        }
        System.out.println(stack);
    }
}

File: dz/s.cpp
#include <iostream>
#include <string>
#include <stack>

using namespace std;

int main(){

    stack<int> s;
    for(int i = 0 ; i < 5; i ++){
        s.push(i);
        cout << i << endl;
    }

    return 0;
}